docs/datasets.md

```diff
@@ -196,7 +216,7 @@ The system architecture of webKnossos allows for versatile deployment options wh
 This may be useful when dealing with large datasets that should remain in your data center.
 [Please contact us](mailto:hello@webknossos.org) or [write a post](https://forum.image.sc/tag/webknossos), if you require any assistance with your setup.
 
-scalable minds also offers a dataset alignment tool called *Voxelytics Align*.
+scalable minds also offer a dataset alignment tool called *Voxelytics Align*.
```

Comment:
I thought we allways used SCM used as a singular subject, even though it's "minds"? I might mistaken, though.